src/platform/web/ui/css/themes/element/theme.css

```diff
@@ -725,6 +725,10 @@ button.link {
     background-color: white;
     list-style: none;
     margin: 0;
+    width: calc(100% - 48px);
```

Comment:
hmm, wouldn't this give very wide menus on bigger screens. If anything, this should be `max-width`? Do we need it at all though?

Reply:
Ok that's true, did not think about wide screens.

Leaving out a width in general but leaving the `flex-wrap: wrap` from [here](https://github.com/vector-im/hydrogen-web/pull/487/commits/6fd2cc24f88ff3a34b189793962fd6d2c5f76b7f) would leave the menu really narrow:

![Screenshot from 2021-10-01 13-02-38](https://user-images.githubusercontent.com/476060/135610661-ce7b40a6-2d2b-4049-92a1-7681f20795cb.png)

I guess what we want is a `min-width` instead. What do you think:

![Screenshot from 2021-10-01 13-06-47](https://user-images.githubusercontent.com/476060/135610774-39badb6d-20c8-4c0d-8753-6ee00e89d436.png)